ui/plugins/auto_classification/main.html

```diff
@@ -39,11 +39,13 @@
           <select
              ng-if="line.type === 'structured' && line.updateText.length > 1"
              ng-model="line.updateSelectOption"
+             title="{{loggedIn() ? '' : 'Must be logged in to change classifications'}}"
+             ng-disabled="!loggedIn()"
              ng-options="value for (key, value) in line.updateText"
              ></select><br>
           <button class="btn btn-xs btn-default"
-                  title="Select best match and click here"
-                  ng-disabled="!line.canSave"
+                  title="{{loggedIn() ? 'Select best match and click here' : 'Must be logged in to change classifications\n'}}{{line.canSave ? '' : 'No changes to save'}}"
```

Comment:
So you'll be printing double here. I would think that the logged in status would be most significant here, so let's print that first. Something like this should do the trick:

```
title="{{loggedIn() ? (line.canSave ? 'Select best match' : 'No changes to save') : 'Not logged in'}}
```